draft-ietf-dnsop-svcb-https.md

```diff
@@ -770,8 +770,30 @@ ALPN IDs that do not uniquely identify a protocol suite (e.g. an ID that
 can be used with both TLS and DTLS) are not compatible with this
 SvcParamKey and MUST NOT be included in the ALPN set.
 
+To establish a connection to the endpoint, clients MUST
+
+1. Identify all supported IDs in the ALPN set.
```

Comment:
"Supported IDs" seems to be ambiguous here.  Needs clarification that this is "supported by the client".

Reply:
Done